### PR TITLE
Fix feeds definition and data feeds OCR to use it.

### PR DESCRIPTION
**Pull request by nolag**

### CODE REVIEW

File: pkg/capabilities/cli/cmd/testdata/fixtures/capabilities/referenceaction/action_builders_generated.go [hunk 1]
```diff
@@ -41,14 +41,14 @@ type someOutputs struct {
...
-	return workflows.AccessField[SomeOutputs, string](c.CapDefinition, "AdaptedThing")
+	return workflows.AccessField[SomeOutputs, string](c.CapDefinition, "adapted_thing")
 }
 
 func NewSomeOutputsFromFields(
 	adaptedThing workflows.CapDefinition[string]) SomeOutputsCap {
 	return &simpleSomeOutputs{
 		CapDefinition: workflows.ComponentCapDefinition[SomeOutputs]{
-			"adaptedThing": adaptedThing.Ref(),
+			"adapted_thing": adaptedThing.Ref(),
```
Comment (nolag) [inline pkg/capabilities/cli/cmd/testdata/fixtures/capabilities/referenceaction/action_builders_generated.go:51]: This was a bug before, found it when I was comparing the output of the richer "not streams trigger"

File: pkg/capabilities/triggers/streams/streams_trigger-schema.json [hunk 2]
```diff
@@ -8,49 +8,91 @@
       "pattern": "^0x[0-9a-f]{64}$",
       "description": "The ID of the data feed."
     },
-    "Feed": {
+    "SignersMetadata": {
```
Comment (nolag) [inline pkg/capabilities/triggers/streams/streams_trigger-schema.json:11]: This is the main file to review

File: pkg/capabilities/triggers/streams/streams_trigger-schema.json [hunk 3]
```diff
@@ -8,49 +8,91 @@
...
           "type": "string",
-          "contentEncoding": "base64",
-          "description": "Full report represented as bytes encoded as base64 string."
+          "format": "base64"
         },
-        "reportContext": {
+        "ReportContext": {
           "type": "string",
-          "contentEncoding": "base64",
-          "description": "Report context represented as bytes encoded as base64 string. This is required to validate the signatures."
```
Comment (DeividasK) [inline pkg/capabilities/triggers/streams/streams_trigger-schema.json:25]: Why are these removed?

**Review by DeividasK — DISMISSED** [review]

**Review by DeividasK — APPROVED** [review]

**Review by bolekk — APPROVED** [review]
